services/identity/service.go

```diff
@@ -39,14 +43,14 @@ func init() {
 // Service handles identities
 type Service struct {
 	*sda.ServiceProcessor
-	identities      map[string]*storage
+	Identities      map[string]*Storage
 	identitiesMutex sync.Mutex
```

Comment:
There's ways to make it work right (having a struct that contains both the identities and the lock and saving only that struct), but you think it's not the most urgent think, then fine, I won't push for it ;)